shared/profile/edit-avatar/index.desktop.tsx

```diff
@@ -297,6 +299,140 @@ class EditAvatar extends React.Component<Props, State> {
   }
 
   render() {
+    if (flags.teamsRedesign) {
+      return (
+        <Kb.Modal
+          onClose={this.props.onClose}
+          header={{
+            leftButton: this.props.createdTeam ? (
+              <Kb.Icon type="iconfont-arrow-left" onClick={this.props.onClose} />
+            ) : null,
+            rightButton: this.props.createdTeam ? (
+              <Kb.Button
+                label="Skip"
+                mode="Secondary"
+                onClick={() => undefined}
+                style={styles.skipButton}
+                type="Default"
+              />
+            ) : null,
+            title: (
+              <ModalTitle
+                teamname={this.props.teamname ? this.props.teamname : ''}
+                title="Upload an avatar"
+              />
+            ),
+          }}
+          allowOverflow={true}
+          footer={{
+            content: (
+              <Kb.Button
+                fullWidth={true}
+                label="Continue"
+                onClick={this.onSave}
+                disabled={!this.state.hasPreview}
+              />
+            ),
+          }}
+        >
+          {!!this.props.error && (
+            <Kb.Banner color="red">
+              <Kb.BannerParagraph bannerColor="red" content={this.props.error} />
+            </Kb.Banner>
+          )}
+          <Kb.Box
+            className={Styles.classNames({dropping: this.state.dropping})}
+            onDragLeave={this.onDragLeave}
+            onDragOver={this.onDragOver}
+            onDrop={this.onDrop}
+            style={Styles.collapseStyles([
+              styles.container,
+              {
```

Comment:
try to avoid inline styles like this. Probably you want something like `collapseStyles([styles.container, this.props.createdTeam && styles.paddingTopForCreatedTeam])` and then define it in the style sheet.

This to make the part of the code where the components and stuff are shorter and thus more readable.